src/data/architecture-and-paradigmes.js

```diff
@@ -7,7 +7,7 @@ export const architectureCollection = {
 				{
 					name: "Programming Paradigms",
 					description: {
-						text: "Learn how programming languages can be categoried into different paraigmes based on their features.",
+						text: "Learn how programming languages can be categoried into different paradigmes based on their features.",
```

Comment:
Don't forget to remove the "e" in "paradigmes"

Reply:
I've made the changes. Please check

Reply:
That looks good, although I think you forgot:
```
Progressive Webapps > Security > Clickjacking
Change "these kind of" to "these kinds of".
```
Also, README.md has these same errors, so I guess the changes should be copied over to there as well. It doesn't necessarily have to be part of the same PR, though.

Reply:
I have done the change, and as for the README.md I'll create another issue which can be done with a new PR.